src/data/aktivitetskrav/forhandsvarselTexts.ts

```diff
@@ -11,25 +11,25 @@ export const sendForhandsvarselTexts = {
   unngaStansInfo: {
     header: "Du kan unngå stans av sykepenger",
     tiltak1:
-      "Kommer du helt eller delvis tilbake i arbeid, oppfyller du aktivitetsplikten og kan fortsatt få sykepenger. Aktivitet kan dokumenteres med gradert sykmelding eller i søknaden",
+      "Kommer du helt eller delvis tilbake i arbeid, oppfyller du aktivitetsplikten og kan fortsatt få sykepenger. Aktivitet kan dokumenteres med gradert sykmelding eller i søknaden om sykepenger.",
     tiltak2:
-      "Gir arbeidsgiveren din en skriftlig begrunnelse for at det ikke er mulig å legge til rette for at du kan jobbe, kan utbetalingen av sykepenger fortsette.",
+      "Aktivitetsplikten vil også være oppfylt hvis du deltar i et arbeidsrettet tiltak i regi av NAV. Ta kontakt med NAV hvis du tenker at dette er aktuelt for deg.",
     tiltak3:
-      "Går det fram av sykmeldingen at du er for syk til å arbeide, får du fortsatt sykepenger.",
+      "Du kan få unntak fra aktivitetsplikten dersom arbeidsgiveren din gir en skriftlig begrunnelse for at det ikke er mulig å legge til rette for deg at du kan jobbe, eller dersom din lege dokumenterer at medisinske grunner klart er til hinder for arbeidsrelatert aktivitet.",
```

Comment:
Du kn godt fjerne `deg` herfra.

`... en skriftlig begrunnelse for at det ikke er mulig å legge til rette for at du kan jobbe..`